silverback/_cli.py

```diff
@@ -32,12 +32,14 @@
 )
 
 if TYPE_CHECKING:
+    from typing import Dict, List
```

Comment:
In Python 3.9+ you can just use `dict` and `list` in type hints, you don't need this